src/tools/perf/perftest.c

```diff
@@ -615,6 +618,7 @@ static ucs_status_t init_test_params(perftest_params_t *params)
     params->super.max_iter          = 1000000l;
     params->super.max_time          = 0.0;
     params->super.report_interval   = 1.0;
+    params->super.pctl_rank         = 50.0;
```

Comment:
maybe rename 'pctl_rank' to 'percentile' ?

Reply:
Percentile is the actual measurement (eg. 1.2 us), percentile rank is the percentage we are aiming for (eg. 50%), so I think this variable name makes sense.

Reply:
@lyu WDYT? we prefer to not use short forms of words because it's not straightforward to parse

Reply:
maybe percentil_rank?

Reply:
Fixed